src/setup/askForDocker/askForDocker.ts

```
@@ -0,0 +1,21 @@
+import inquirer from 'inquirer';
+
+export const askForDocker = async (): Promise<string> => {
+  const answers = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'dockerAppPort',
+      message: 'Enter the port to expose Docker (default: 4321):',
+      default: 4321,
+      validate: (input: string) => {
+        const port = Number(input);
+        if (isNaN(port) || port < 1024 || port > 65535) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)